OWNERS_ALIASES

```diff
@@ -468,14 +449,7 @@ aliases:
     - msau42
     - jsafrane
     - xing-yang
-  sig-security-approvers:
```

Comment:
The sig-security aliases need to be maintained.

Reply:
I'm happy to ack a PR restoring the alias if needed, but can you add a comment in the file explaining why it's present so it doesn't get removed again (the alias isn't used in k/k and the sig-security charter says it doesn't directly own Kubernetes component code)

Reply:
oh, ha... just realized this hasn't merged yet. can you provide the comment content about why the alias is needed?

Reply:
@tabbysable let me get this in now (so reduce chances for rebasing). let's open a fresh PR to add things back as needed. (please see @liggitt 's questions above)